code/game/machinery/computer/arcade.dm

```diff
@@ -998,7 +998,7 @@
 	to_chat(user, "<span class='warning'>You flip the switch on the underside of [src].</span>")
 	active = 1
 	visible_message("<span class='notice'>[src] softly beeps and whirs to life!</span>")
-	playsound(src.loc, 'sound/machines/defib_SaftyOn.ogg', 25, 1)
+	playsound(src.loc, 'sound/machines/defib_saftyOn.ogg', 25, 1)

```

Comment:
Think it's worth fixing the typo in the sound name?
`safety`

Reply:
nah they are both like that..its obscure file naming short hand. Should see me naming vars in code sometimes.

Reply:
Fair enough hahaha